app/models/invoice.rb

```diff
@@ -47,6 +47,25 @@ class Invoice < ActiveRecord::Base
 
   private
 
+  def self.period(period)
+    case period
+    when 'current-month'
+      where('invoices.ends_at >= ? AND invoices.ends_at <= ?', Time.now.beginning_of_month, Time.now)
```

Comment:
@yunixon another thing: if you set this fields as `date` in database schema, why you use time in conditions? Maybe it will be better to use Date too? like `where('invoices.ends_at between ? AND ?', Date.current.beginning_of_month, Date.current.end_of_month)`